src/browser/BrowserService.cpp

```diff
@@ -513,8 +513,16 @@ QList<Entry*> BrowserService::sortEntries(QList<Entry*>& pwEntries, const QStrin
     const QString baseSubmitUrl = url.toString(QUrl::StripTrailingSlash | QUrl::RemovePath | QUrl::RemoveQuery | QUrl::RemoveFragment);
 
     QMultiMap<int, const Entry*> priorities;
-    for (const Entry* entry : pwEntries) {
-        priorities.insert(sortPriority(entry, host, submitUrl, baseSubmitUrl), entry);
+    for (Entry* entry : pwEntries) {
+        const int priority = sortPriority(entry, host, submitUrl, baseSubmitUrl);
+
+        // Ignore matches that aren't exact enough
+        if (BrowserSettings::bestMatchOnly() && priority < 100) {
```

Comment:
Does 100 represent percent matching? If not, this might need some explaining.

Reply:
Not exactly. It's from the old HTTP implementation that tries to mimic a percentage. See https://github.com/varjolintu/keepassxc/blob/e0a06bd515d77a9b62991a5ee2a8a4740cf79bb3/src/browser/BrowserService.cpp#L650

Reply:
Interesting... This will be a target for 2.4 haha